### ISSUE TITLE
Broken tests on commit a105e3

### ISSUE THREAD

**Issue by lintool** [gwf-uwaterloo/waterwheel#24]
I get:

```
$ python -m unittest
..FF.
======================================================================
FAIL: test_conjunctions (tests.test_waterwheel.TestWaterWheel)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/Users/jimmylin/workspace/waterwheel/tests/test_waterwheel.py", line 43, in test_conjunctions
    self.assertEqual(str(doc.ents[3]), 'Saint Lawrence River')
AssertionError: 'Saint Lawrence' != 'Saint Lawrence River'
- Saint Lawrence
+ Saint Lawrence River
?               ++++++


======================================================================
FAIL: test_issue20 (tests.test_waterwheel.TestWaterWheel)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/Users/jimmylin/workspace/waterwheel/tests/test_waterwheel.py", line 58, in test_issue20
    self.assertEqual(str(doc.ents[1]), 'Great Slave Lake')
AssertionError: 'Great Slave' != 'Great Slave Lake'
- Great Slave
+ Great Slave Lake
?            +++++


----------------------------------------------------------------------
Ran 5 tests in 4.396s

FAILED (failures=2)
```

**Comment by Govind9** [comment]
Submitted the wrong file before. PL sent with correct file.

**Comment by lintool** [comment]
Fixed with 4234c82
